cypress/support/commands.js

```diff
@@ -23,3 +23,53 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add("getTestData", () => { 
+    let testUser = Cypress.env('testUser');
+    let testUserMail = Cypress.env('testUserMail');
+    let testPwd = Cypress.env('testPwd');
+    return {testUser: testUser, testUserMail: testUserMail, testPwd: testPwd};
+});
+
+Cypress.Commands.add("getChangePwdData", () => {
+    let changePwdUserMail = Cypress.env('changePwdUserMail');
+    let changePwdPwd = Cypress.env('changePwdPwd');
+    let newPwd = Cypress.env('newPwd');
+    return {changePwdUserMail: changePwdUserMail, changePwdPwd: changePwdPwd, newPwd: newPwd};
+});
+
+Cypress.Commands.add("reLogin", (userMail, userPassword) => { 
+    cy.visit("http://localhost:3000/signout");
+    cy.visit("http://localhost:3000/signin");
+    cy.get("input[name=email]").type(`${userMail}`);
+    cy.get("input[name=password]").type(`${userPassword}`);
+    cy.get("button[type=submit]").click();
+});
+
+Cypress.Commands.add("openAppDrawer", () => { 
+    cy.get("button[data-cy=appDrawerOpener]").click();
+    cy.get("div[data-cy=appDrawerDiv]").should('be.visible');
+});
+
+Cypress.Commands.add("navigateToChangePasswordForm", () => {
+    cy.openAppDrawer();
+    cy.get("a[data-cy=changePasswordDrawerButton]").last().click();
+});
+
+
+Cypress.Commands.add("navigateToProductsPage", () => {
+    cy.openAppDrawer();
+    cy.get("a[data-cy=productsDrawerButton]").last().click();
+});
+
+Cypress.Commands.add("navigateToInvitePage", () => {
+    cy.openAppDrawer();
+    cy.get("a[data-cy=inviteDrawerButton]").last().click();
+});
+
+//https://github.com/cypress-io/cypress/issues/761
+Cypress.Commands.add("cleanUpXHR", function() {
+    if (Cypress.env("run_all_suite")) {
+      cy.visit(`${Cypress.env("yourAppUrl")}/404`, { failOnStatusCode: false });
```

Comment:
Need to use the correct environment variable

Reply:
Reading the issue, it looks like it's been fixed in 3.1.4? Were you still seeing the problem?

Reply:
Yeah issue says it's fixed but I still get all the XHRs in previous tests open. But I'm not sure if it's cuz of this issue or it's the issue with Firestore XHRs not being closed we had earlier (starting with @bfirsh mentioning in his initial #292)  I haven't found a solution for it, I just decided to ignore it and implement those tests anyway. XHRs stayed open also in another Git repository with cypress and firestore I've tested.

This `cleanUpXHR` is currently not used, I just kept it in the code for us to discuss during the next call.